lib/benchpark/experiment.py

```diff
@@ -77,7 +77,7 @@ def compute_config_variables_wrapper(self):
 class SingleNode:
     variant(
         "single_node",
-        default=True,
+        default=False,
```

Comment:
@rfhaque This fixes the issue for `hpl+strong` and some other experiments that check for multiple scaling options (https://github.com/LLNL/benchpark/actions/runs/14068780147/job/39397789837), but means that it is now possible for no scaling option to be specified.

Reply:
Being addressed in #714